ClientApp/src/app/not-found-page/not-found-page.component.html

```diff
@@ -0,0 +1,9 @@
+<h2 *ngIf="this.notFoundReason === this.NotFoundReason.Unknown">Invalid URL!</h2>
```

Comment:
big fan of this! It's nice to know why an error has occurred and this avoids copy-pasting needless code